### PR TITLE
Created entities, repositories, TourService and a TourScheduler

### PR DESCRIPTION
**Pull request by KamilM02**
TourManagement microservice

### CODE REVIEW

File: Microservices/TourManagementService/src/main/java/com/example/TourManagementService/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @Column(nullable = false)
+    private String password;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private UserType userType;
+
+    // Getters and Setters
+
+    public enum UserType {
```
Comment (carthy27) [inline Microservices/TourManagementService/src/main/java/com/example/TourManagementService/entity/User.java:26]: Dont think we are going ahead with admin could probably delete admin from the enums.

File: Microservices/TourManagementService/src/main/java/com/example/TourManagementService/entity/Tour.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    public void setEmailId(String emailId) {
+        this.emailId = emailId;
+    }
+
+    public String getTourDateTime() {
+        return tourDateTime;
+    }
+
+    public void setTourDateTime(String tourDateTime) {
```
Comment (carthy27) [inline Microservices/TourManagementService/src/main/java/com/example/TourManagementService/entity/Tour.java:55]: Are we okay with this just being a string?

**Review by carthy27 — CHANGES_REQUESTED** [review]
Nice work looks good. One enum change and a question about how the formatting of our time is.

**Review by carthy27 — APPROVED** [review]

**Review by BrenyQT — APPROVED** [review]
lgtm

**Pull request merged**